tools/SetupFlow/DevHome.SetupFlow/ViewModels/PackageViewModel.cs

```diff
@@ -225,6 +259,47 @@ private string GetPackageDescription()
         // Source
         return CatalogName;
     }
+
+    /// <summary>
+    /// Indicates if a specific version of the package can be selected to install
+    /// </summary>
+    /// <returns>True a package version can be specified to install</returns>
+    private bool IsVersioningSupported()
+    {
+        // Store packages have a single version
+        return !_wpm.IsMsStorePackage(_package);
+    }
+
+    /// <summary>
+    /// Checks if the package is selectable
+    /// </summary>
+    /// <returns>True if the package is selectable</returns>
+    /// <remarks>Allow selecting a different version to install if the package is installed</remarks>
+    private bool IsSelectable()
+    {
+        if (IsInstalled)

```

Comment:
Consider turning this into an early return with `if(!IsInstalled) {return true}` to reduce indentation level.